### PR TITLE
[ICD] Add OperatingMode attribute to the LIT-ICD-app

### PR DESCRIPTION
**Pull request by mkardous-silabs**
#### Description
When the Operating mode attribute was added to the xml, is not enable in the lit-icd-app.
PR adds :
- Operating mode attribute to the LIT ICD app
- Adds associated test

#### Tests
Add integration test to validate attribute.

### CODE REVIEW

**Review by yunhanw-google — APPROVED** [review]

**Review by jmartinez-silabs — APPROVED** [review]

**Review by saurabhst — APPROVED** [review]

**Review by andy31415 — APPROVED** [review]

File: src/app/tests/suites/TestIcdManagementCluster.yaml [hunk 1]
```diff
@@ -52,13 +52,25 @@ tests:
                   },
               ]
 
+    - label: "Verify the ICD is operating as a LIT ICDCounter"
```
Comment (bzbarsky-apple) [inline src/app/tests/suites/TestIcdManagementCluster.yaml:55]: What is a "LIT ICDCounter"?

File: src/app/tests/suites/TestIcdManagementCluster.yaml [hunk 2]
```diff
@@ -52,13 +52,25 @@ tests:
...
+          value: 1
+
     - label: "Unregister Client Registered During Commissioning"
       command: "UnregisterClient"
       arguments:
           values:
               - name: "CheckInNodeID"
                 value: commissionerNodeId
 
+    - label: "Verify the ICD is operating as a LIT ICDCounter"
```
Comment (bzbarsky-apple) [inline src/app/tests/suites/TestIcdManagementCluster.yaml:68]: This label does not look right.

File: src/app/tests/suites/TestIcdManagementCluster.yaml (hunk 1 above)
Comment (mkardous-silabs) [reply src/app/tests/suites/TestIcdManagementCluster.yaml:55]: a really big typo

Comment (mkardous-silabs) [reply src/app/tests/suites/TestIcdManagementCluster.yaml]: updated

File: src/app/tests/suites/TestIcdManagementCluster.yaml (hunk 2 above)
Comment (mkardous-silabs) [reply src/app/tests/suites/TestIcdManagementCluster.yaml]: updated

**Review by bzbarsky-apple — APPROVED** [review]

**Pull request merged**